src/App.jsx

```diff
@@ -73,17 +190,30 @@ class App extends Component {
             path="/user/:profileId"
             render={props => <ProfilePage userId={userId} {...props} />}
           />
+          <Route
+            path="/notificationpanel"
+            exact
+            render={() => <NotificationPanel notifications={notifications} />}
+          />
         </Router>
       );
     }
 
     return (
       <div className="App">
+        <div id="overlay" />
```

Comment:
I don't like the use of overlay like this. Make this part of the notification panel element so that everything is encapsulated within a single component.

I think it should look something like there is a notification-panel-wrapper component that is actually the size of the whole screen and most of it has a background color of black (with reduced opacity) and the actual notification panel is within this wrapper.

Reply:
pretty much, there should generally be as few changes to app.jsx and app.css as possible. the other code you added to app.jsx is necessary, but this specifically is not

Reply:
oops this code wasn't needed at all, i had already moved it to notification panel